src/components/ChatApp/MessageComposer.react.js

```diff
@@ -12,13 +12,16 @@ import Modal from 'react-modal';
 import ReactFitText from 'react-fittext';
 import Close from 'material-ui/svg-icons/navigation/close';
 import TextareaAutosize from 'react-textarea-autosize';
-import './ChatApp.css'
+import YTSearch from 'youtube-api-search';
+import './ChatApp.css';
+
 injectTapEventPlugin();
 
 let ENTER_KEY_CODE = 13;
 let UP_KEY_CODE = 38;
 let DOWN_KEY_CODE = 40;
 var flag=1;
+const API_KEY = 'AIzaSyB9hQRq9nEns5OMBdSdCRx0o2Bo14IYPVM';
```

Comment:
@anshumanv I myself was thinking about showing the API key is not a good idea, as it is specific to a user account. Can you suggest any way to maybe hide/encrypt it.

Reply:
https://github.com/fossasia/chat.susi.ai/pull/1215#discussion_r189470268

Reply:
But it should be fine for now till we're not in production. Let's see if others find this relevant.